src/test/java/com/xceptance/neodymium/testclasses/data/annotation/InstantiateDtoViaJsonPathInAnnotation.java

```diff
@@ -0,0 +1,23 @@
+package com.xceptance.neodymium.testclasses.data.annotation;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.module.statement.testdata.Data;
+import com.xceptance.neodymium.util.DataUtils;
+
+@RunWith(NeodymiumRunner.class)
+public class InstantiateDtoViaJsonPathInAnnotation
+{
+    @Data("$.user")
```

Comment:
Could you add another test retrieving an object from a more complex structure?

e.g `"$.user.payment.creditcards[2]"`

Reply:
To avoid introducing another data structure I used ` "$.user.contacts.friends[2]"`. Done